mocks/service/launchpad/index.js

```diff
@@ -29,12 +29,18 @@ router.get('/devel/*snaps', (req, res) => {
         {
           resource_type_link: `${base}/#snap`,
           git_repository_url: `${prefix}mock-repo-1`,
-          self_link: `${base}/${owner}/+snap/mock-snap-1`
+          self_link: `${base}/${owner}/+snap/mock-snap-1`,
+          name: 'mock-snap-1'
         },
         {
           resource_type_link: `${base}/#snap`,
           git_repository_url: `${prefix}mock-repo-2`,
-          self_link: `${base}/${owner}/+snap/mock-snap-1`
+          self_link: `${base}/${owner}/+snap/mock-snap-2`,
+          name: 'mock-snap-2'
+        },
+        {
+          resource_type_link: `${base}/#snap`,
+          git_repository_url: `${prefix}mock-repo-3`
```

Comment:
Well.  I should clarify.  A snap where BSI has worked out the snap name will have the `store_name` field set; that's up to BSI to fill in.  From Launchpad's point of view there's nothing to say that it must be registered, although I expect once we complete the integration we'll be able to guarantee that it will be.

From the mock point of view, though, filling in as much as we have so far is just fine.